src/controllers/pettype/add.ts

```diff
@@ -0,0 +1,21 @@
+import { type Request, type Response, type NextFunction } from 'express'
+import addType from '../../queries/pettype/add'
+import { type IPetType } from '../../interfaces/fakeDataTypes'
+import { validateTitle } from '../../validation'
+
+const createType = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
since this might understood as create user type we need to be more specific
```suggestion
const createPetType = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```